index.js

```diff
@@ -3825,6 +3825,32 @@ export const createRubyBom = async (path, options) => {
   return {};
 };
 
+const removeDuplicates = (pkgList, dependencies) => {
```

Comment:
We have trimComponents and mergeDependencies for removing duplicates? Can we use them instead?

Reply:
@prabhu MergeDependencies could maybe work if the newDependencies parameter was made optional. However, might have unexpected results given cdxgen is having issues identifying the parentComponent for many nodejs and dotnet projects. trimComponents I think would work for pkgList.